crates/turborepo-lib/src/cli.rs

```diff
@@ -309,6 +322,11 @@ pub struct RunArgs {
     /// .html). Outputs dot graph to stdout when if no filename is provided
     #[clap(long, num_args = 0..=1, default_missing_value = "")]
     pub graph: Option<String>,
+    /// Environment variable mode.
+    /// Loose passes the entire environment.
+    /// Strict uses an allowlist specified in turbo.json.
+    #[clap(long = "env", default_value = "infer", num_args = 0..=1, default_missing_value = "infer")]
```

Comment:
Smallest nit, but do we want to consider naming this `--env-mode`? I feel like `--env` could be used to forward environment variables to tasks, like in Docker.

Reply:
Naming is precisely the bikesheds I want to address in this review, actually!

Reply:
Update this to `--experimental-env-mode` for now and hid it from help output. We can bikeshed before real release